raiden-dapp/src/components/TokenOverlay.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <v-overlay :value="show" absolute opacity="1.0" class="token-network-overlay">
+    <v-container class="container">
+      <v-row no-gutters justify="end">
+        <v-btn @click="cancel" icon class="close-button">
+          <v-icon>mdi-close</v-icon>
+        </v-btn>
+      </v-row>
+
+      <v-row id="connectNew">
+        <v-col cols="12">
+          <v-list class="item-list">
+            <v-list-item @click="navigateToTokenSelect()">
+              <v-col cols="2">
+                <v-list-item-avatar>
+                  <v-btn class="mx-2" fab dark small color="primary">
+                    <v-icon dark large>mdi-plus</v-icon>
+                  </v-btn>
+                </v-list-item-avatar>
+              </v-col>
+              <v-col cols="10" align-self="center" class="connect-new-token">
+                {{ $t('tokens.connect-new') }}
+              </v-col>
+            </v-list-item>
+          </v-list>
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col cols="2" align-self="center"></v-col>
+        <v-col cols="10" align-self="center" class="header">
+          {{ $t('tokens.connected.header') }}
+        </v-col>
+      </v-row>
+
+      <v-row id="tokenList">
+        <v-col cols="12">
+          <v-list v-for="(token, i) in tokens" :key="i" class="item-list">
```

Comment:
in most of the components, we follow http://getbem.com/ naming.  Can we make sure that this complies to the same naming scheme for consistency?

Reply:
Any specific reason we use BEM? Seems super verbose and get's ugly quickly when nested in SASS. But will change

Reply:
Sure the main reason is clarity and uniqueness of the elements when we are editing CSS. Sure it is verbose but that is the point. While the component scoping might work for the page itself, it's harder to figure out which rule goes where. This becomes even more obvious when you have a lot of components and maybe also some global style in the mix using the same class. So far I am happier using BEM that I was dealing with CSS before introducing it.

Reply:
Yeah but the verboseness and DOM readability could easily be improved by using CSS modules. That way you get the best of both worlds without having any compromises. BEM solves the global CSS scope problem but has a lot of drawbacks. I will do the change for now but we should rethink this in the future